discovery/test/unit_tests/test_wordnet_pearson_matching.py

```diff
@@ -0,0 +1,31 @@
+"""
+Ensure that the WordNet and Pearson matching algorithms complete without errors
+"""
+import os
+import logging.config
+
+from _pytest.fixtures import fixture
+from discovery import Discovery
+from utils.datagen import FakeDataGen
+from utils.dataframe_matcher import DataFrameMatcher
+
+logger = logging.getLogger(__name__)
+
+
+@fixture
+def discovery_instance():
+    return Discovery({})
+
+
+def test_wordnet_pearson_matching(discovery_instance, tmp_path_factory):
+    fake_data = FakeDataGen()
+    temp_path = tmp_path_factory.mktemp("mock_fs")
```

Comment:
Will the test_fs get cleaned up automatically? Can multiple test runs conflict with each other?

Reply:
It's a temp folder